content/articles/2018-08-scales-1-0-0.Rmarkdown

```diff
@@ -0,0 +1,98 @@
+
+Install the latest version with:
+```{r, eval = FALSE}
+install.packages("scales")
+```
+
+# New features
+## Formatter changes
+
+Thanks to the help of [@larmarange](https://github.com/larmarange), scales has 
+now added a generic formatter function `number_format()` that powers the majority
+of the formatters in the package. 
+
+This change altered the arguments of most of the formatters: 
+`comma_format()`, `percent_format()`, `unit_format()`, `dollar_format()`,
+`ordinal_format()`, and `scientific_format()` all gained new arguments allowing 
+users to specify rounding accuracy, a scaling value, a prefix, and a suffix to 
+customize output. Furthermore, all of these functions now allow user 
+specification of the thousands separator, `big.mark`, and the decimal separator,
+`decimal.mark`. Users of these formatters will notice that the default thousands
+separator is now a space. This default was chosen as a useful compromise for
+an international audience (a space for thousands separator and a dot for decimal 
+separator) and is officially endorsed by SI/ISO 31-0 standard, as well as by the 
+International Bureau of Weights and Measures and the International Union of 
+Pure and Applied Chemistry (IUPAC), the American Medical Association's widely 
+followed AMA Manual of Style, and the Metrication Board, among others 
+([read more here](https://www.wikiwand.com/en/Decimal_separator#/Digit_grouping)).
+Those interested in number formatting with no separator can simply define 
+`big.mark = ""` to revert to the previous behaviour. 
+
+```{r}
+library(scales)
+number(c(12.3, 4, 12345.789, 0.0002))
+
+# these functions round by default, but you can set the accuracy
+number(c(12.3, 4, 12345.789, 0.0002), big.mark = "", accuracy = .01)
+```
+
+Finally, three additional formatters have been added and `ordinal_format()` has
+gained new rules for French and Spanish formatting. The new `pvalue_format()` formats p-values, `number_bytes_format()` formats numeric vectors into byte measurements, and 
+`time_format()` provides support for formatting POSIXt and hms objects.
+
+## New transformations
+
+Two new transformations were added to the package for this release: `psuedo_log_trans()`, 
+and `modulus_trans()`. `pseudo_log_trans()` transforms data on a signed 
+logarithmic scale with a smooth transition to a linear scale around 0. The 
+`modulus_trans()` was added along with a refactored `boxcox_trans()` to 
+provide a better option for negative numbers. Both `modulus_trans()` and 
+`boxcox_trans()` gained an argument `offset` which now allows users to fit both
+type 1 and type 2 Box-Cox transformations. 
+
+## Better breaks
+
+In a long awaited fix, `log_breaks()` now returns integer multiples of 
+integer powers of base when finer breaks are needed on the log scale.
+This will change all ggplot graphics with log-transformed axes. 
+```{r, load, echo = FALSE}
+suppressWarnings(suppressPackageStartupMessages(library(dplyr)))
```

Comment:
Double check with @hadley here, but I think the convention has been to load all of the libraries at the beginning.